docs/sphinx/data_center_install.rst

```diff
@@ -253,12 +268,19 @@ command to build CUDA Quantum:
     :start-after: [>CUDAQuantumBuild]
     :end-before: [<CUDAQuantumBuild]
 
-The CUDA Quantum build will compile or omit optional components automatically depending
-on whether the necessary pre-requisites are found in the build environment.
-Please check the build log to confirm that all desired components have been built. If
-you see a message that a component has been skipped, make sure you followed the 
-instructions for installing the necessary prerequisites and build dependencies, 
-and have set the necessary environment variables as described in this document.
+Please check the build log to confirm that all desired components have been built. 
```

Comment:
Where can they find the build log and what is its name?

Reply:
The build log is printed to the console when you execute the command above.

Reply:
rephrased